packages/perseus/src/validation.types.ts

```diff
@@ -180,7 +180,14 @@ export type PerseusSorterUserInput = {
     changed: boolean;
 };
 
-export type PerseusTableRubric = PerseusTableWidgetOptions;
+export type PerseusTableRubric = {
```

Comment:
I "exploded" this type from `PerseusTableWidgetOptions` because `headers` aren't needed for validation or scoring. TypeScript will still ensure that objects of one sort are compatible with the other.

Reply:
I don't think the other fields are used, so might be worth a try to remove them. You'd also have to remove them from the tests.

```suggestion
    // Translatable Text; A 2-dimensional array of text to populate the table with
    answers: ReadonlyArray<ReadonlyArray<string>>;
};
```